packages/e2e/specs/overview/display-initial-os-version.e2e.ts

```diff
@@ -0,0 +1,26 @@
+import OverviewPage from "../../page-objects/overview.page"
+
+describe("Overview Page", () => {
+  describe("Have initial OS version", () => {
+    before(async () => {
+      // Waiting for device connected through USB
+      await browser.executeAsync((done) => {
+        setTimeout(done, 10000)
+      })
+    })
+
+    it("should have valid OS version", async () => {
+      const VERSION_REGEXP = /[0-9].[0-9].[0-9]/
+      const HARDCODED_VERSION = process.env.TEST_INITIAL_VERSION
```

Comment:
I'm wondering if some fixture/script shouldn't set the initial version? 🤔

Reply:
The initial version will be flashed to Pure with `pureflash` script. And this variable is needed just to, not change the code if for some reason we decided to bump the default version